PennMobile/Assets.xcassets/Color/UI Palette/uiGroupedBackgroundSecondary.colorset/Contents 2.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Something funky going on here -- all of these files are duplicates... we will figure this out at the meeting. I was having similar problems last week